ChromePhp.php

```diff
@@ -377,7 +382,35 @@ protected function _addRow(array $logs, $backtrace, $type)
 
     protected function _writeHeader($data)
     {
-        header(self::HEADER_NAME . ': ' . $this->_encode($data));
+        $header = self::HEADER_NAME . ': ' . $this->_encode($data);
+        // Most HTTPD servers have a default header line length limit of 8kb, must test to avoid 500 Internal Server Error.
+        if (strlen($header) > self::HTTPD_HEADER_LIMIT) {
+            $data['rows'] = [];
+            $data['rows'][] = [
+                [
+                    'ChromePHP Error: The HTML header will surpass the limit of ' .
+                    $this->_formatSize(self::HTTPD_HEADER_LIMIT) . ' (' . $this->_formatSize(strlen($header)) .
+                    ') - You can increase the HTTPD_HEADER_LIMIT on ChromePHP class, according to your Apache ' .
+                    'LimitRequestFieldsize directive'
+                ], '', self::ERROR
+            ];
+            $header = self::HEADER_NAME . ': ' . $this->_encode($data);
+        }
+        header($header);
+    }
+
+    protected function _formatSize($arg) {
```

Comment:
It looks like this method coding style is not same as other methods.

I think we should fix/organize coding style and I suggest we can follow the PSR-2 coding style :).

And the [PHP_CodeSniffer ](https://github.com/squizlabs/PHP_CodeSniffer) is one of coding style check tool we can consider.